OpenRA.Mods.Common/TraitsInterfaces.cs

```diff
@@ -145,7 +145,24 @@ public interface IProductionIconOverlay
 		bool IsOverlayActive(ActorInfo ai);
 	}
 
-	public interface INotifyTransform { void BeforeTransform(Actor self); void OnTransform(Actor self); void AfterTransform(Actor toActor); }
+	public interface INotifyTransform
+	{
+		void BeforeTransform(Actor self);
+		void OnTransform(Actor self);
+		void AfterTransform(Actor toActor);
+	}
+
+	public interface INotifyDeploy
+	{
+		void FinishedDeploy(Actor self);
+		void FinishedUndeploy(Actor self);
+	}
+
+	public interface INotifyDeployAnimation
```

Comment:
It feels like this interface really should be called `INotifyDeploy` or perhaps `ITriggerDeploy`, and the old `INotifyDeploy` be something else.